src/attestation/azext_attestation/tests/latest/test_attestation_scenario.py

```diff
@@ -85,9 +76,11 @@ def step_attestationproviders_delete(test, rg):
              '--resource-group "{rg}" '
              '--yes',
              checks=[])
+    """
     test.cmd('az attestation list '
              '--resource-group "{rg}"',
              checks=[test.check('length(value)', 0)])
+    """
```

Comment:
Here is for checking whether delete actual success, why u comment them out?

Reply:
> Here is for checking whether delete actual success, why u comment them out?

This is because the `list -g` functionality was broken on service side, we decided to bypass the test. Now it has been reverted.